Core/TextureReplacer.cpp

```diff
@@ -744,46 +745,9 @@ float TextureReplacer::LookupReduceHashRange(int& w, int& h) {
 	}
 }
 
-class LimitedWaitable : public Waitable {
-public:
-	LimitedWaitable() {
-		triggered_ = false;
-	}
-
-	void Wait() override {
-		if (!triggered_) {
-			std::unique_lock<std::mutex> lock(mutex_);
-			cond_.wait(lock, [&] { return !triggered_; });
-		}
-	}
-
-	bool WaitFor(double budget) {
-		uint32_t us = budget > 0 ? (uint32_t)(budget * 1000000.0) : 0;
-		if (!triggered_) {
-			if (us == 0)
-				return false;
-			std::unique_lock<std::mutex> lock(mutex_);
-			cond_.wait_for(lock, std::chrono::microseconds(us), [&] { return !triggered_; });
```

Comment:
Oops, I can't believe I didn't notice that.  I'm sure I copied it from Event and changed the line, but even so.

-[Unknown]